api/ImageData.json

```
@@ -98,6 +98,54 @@
           }
         }
       },
+      "colorSpace": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/canvas.html#dom-imagedata-colorspace",
+          "support": {
+            "chrome": {
+              "version_added": "92"
+            },
+            "chrome_android": {
+              "version_added": "92"
+            },
+            "edge": {
+              "version_added": "92"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "78"
+            },
+            "opera_android": {
+              "version_added": "65"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": "92"
+            }
+          },
+          "status": {
+            "experimental": false,
```

Since it's still Chromium-only:

```suggestion
            "experimental": true,
```